ice.c

```diff
@@ -4199,8 +4212,12 @@ static gboolean janus_ice_outgoing_stats_handle(gpointer user_data) {
 					json_object_set_new(info, "nacks-sent", json_integer(stream->component->out_stats.audio.nacks));
 					json_object_set_new(info, "retransmissions-received", json_integer(stream->audio_rtcp_ctx->retransmitted));
 				}
-				janus_events_notify_handlers(JANUS_EVENT_TYPE_MEDIA, JANUS_EVENT_SUBTYPE_MEDIA_STATS,
-					session->session_id, handle->handle_id, handle->opaque_id, info);
+				if(combinedEvent)
+					json_array_append_new(combinedEvent, info);
+				else {
```

Comment:
Just for readability, could you add brackets to the `if` as well, even though the action is simple? Since the `else` has them, it would make code easier to follow. Same for the `if(combinedEvent)` below.